interface/orders/procedure_stats.php

```diff
@@ -331,119 +331,114 @@ function process_result_code($row)
+                    foreach ($arr_by as $key => $value) {
+                        echo "    <option value='" . attr($key) . "'";
+                        if ($key == $form_by) {
+                            echo " selected";
+                        }
+
+                        echo ">" . text($value) . "</option>\n";
+                    }
+                ?>
+            </select>
+        </div>
+        <div class="col">
+            <?php echo xlt('Sex'); ?>:
+        </div>
+        <div class="col">
+            <select class='form-control' name='form_sexes' title='<?php echo xla('To filter by sex'); ?>'>
+                <?php
+                    foreach (array(3 => xl('Men and Women'), 1 => xl('Women Only'), 2 => xl('Men Only')) as $key => $value) {
+                        echo "       <option value='" . attr($key) . "'";
+                        if ($key == $form_sexes) {
+                            echo " selected";
+                        }
+
+                        echo ">" . text($value) . "</option>\n";
+                    }
+                ?>
+            </select>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('Columns'); ?>:
+            <select class='form-control' name='form_show[]' size='4' multiple title='<?php echo xla('Hold down Ctrl to select multiple items'); ?>'>
+                <?php
+                    foreach ($arr_show as $key => $value) {
+                        $title = $value['title'];
+                        if (empty($title) || $key == 'title') {
+                            $title = $value['description'];
+                        }
+
+                        echo "    <option value='" . attr($key) . "'";
+                        if (is_array($form_show) && in_array($key, $form_show)) {
+                            echo " selected";
+                        }
+
+                        echo ">" . text($title) . "</option>\n";
+                    }
+                ?>
+            </select>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('To{{Destination}}'); ?>:
+
+            <?php
+                foreach (array(1 => xl('Screen'), 2 => xl('Printer'), 3 => xl('Export File')) as $key => $value) {
+                    echo "   <input type='radio' name='form_output' value='" . attr($key) . "'";
+                    if ($key == $form_output) {
+                        echo ' checked';
+                    }
+
+                    echo " />" . text($value) . " &nbsp;";
+                }
+            ?>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-4 col-md-4">
+    <div class="col">
+         <button type='submit' class='btn btn-primary btn-save' name='form_submit' value='<?php echo xla('Submit') ?>' title='<?php echo xla('Click to generate the report'); ?>'></button>
```

Comment:
I am pretty sure that won't work (it will likely show an empt button). Let me know if it is testing to work. The button "label" needs to be outside the html attributes (like a link) like you had it previously. The value is just used in the POST setting. So, doing:
```
<button type='submit' class='btn btn-primary btn-save' name='form_submit' value='form_submit' title='<?php echo xla('Click to generate the report'); ?>'><?php echo xlt('Submit') ?></button>
```
will do the following:
1. The label of the button with be "Submit"
2. When click this Submit, it will pass a POST variable with name `form_submit` that has value `form_submit`. This then allows it to pass the conditional on line 444 since the `form_submit` POST value is not empty